packages/report-server/src/reportBuilder/transform/functions/index.ts

```diff
@@ -26,7 +27,7 @@ import {
   paramsValidator as gatherColumnsParamsValidator,
 } from './gatherColumns';
 import { buildFetchData, paramsValidator as fetchDataParamsValidator } from './fetchData';
-import { buildOrderColumns, orderColumnsSchema } from './orderColumns';
+import { buildOrderColumns, paramsValidator as orderColumnsParamsValidator } from './orderColumns';
```

Comment:
Thanks for doing this refactor, all transform validators look neat together now!